test/functional/terminal/window_spec.lua

```diff
@@ -54,9 +55,7 @@ describe(':terminal window', function()
         {3:-- TERMINAL --}                                    |
       ]])
 
-      if iswin() then
-        return  -- win: :terminal resize is unreliable #7007
-      end
+      skip(iswin(), 'win: :terminal resize is unreliable #7007')
```

Comment:
off-topic here, but these tests might work since we got rid of winpty

Reply:
They don't, I tried.